app/mailers/adoption_application_mailer.rb

```diff
@@ -0,0 +1,12 @@
+class AdoptionApplicationMailer < ApplicationMailer
+  def new_adoption_application(adoption_application)
```

Comment:
I think I would rename both the class and this method. It seems like this is more contextual than your class implies. For instance I would choose `NewAdoptionApplicationMailer` which described better what it's purpose is (to send mails when a new application is created). Then I'd probably just call my method `notification`.